tests/ignite/handlers/test_ema_handler.py

```diff
@@ -28,7 +35,10 @@ def _get_dummy_step_fn(model: Union[nn.Module, DataParallel, DistributedDataPara
 
     def step_fn(engine, batch):
         """Increment the weight by 1 at each iteration"""
-        model.weight.data.add_(1)
+        if isinstance(model, (DataParallel, DistributedDataParallel)):
+            model.module.weight.data.add_(1)
+        else:
+            model.weight.data.add_(1)
```

Comment:
Maybe, we can do
```suggestion
        _unwrap_model(model).weight.data.add_(1)
```
?